tests/ssg_test_suite/common.py

```diff
@@ -582,6 +453,10 @@ def fetch_local_test_scenarios(tests_dir, local_env_yaml):
     if os.path.exists(tests_dir):
         tests_dir_files = os.listdir(tests_dir)
         for test_case in tests_dir_files:
+            # Skip vim swap files, they are not relevant and cause Jinja
+            # expansion tracebacks
+            if test_case.endswith(".swp"):
```

Comment:
`vim` is not that special, why not to use this opportunity to introduce a `if file_known_as_useless(test_case):` function that will make it easier to exclude more patterns?